### PR TITLE
Fix how arguments are handled for commands running in terminal

### LINKED ISSUE #3320: Full content of Data field is not sent during Serverless Function Invoke
eg. with Quarkus cloudevents, the sample function responds with the request. Try running the function locally, then invoking with content type `application/json` and the default Data value ("Hello World")

The response contains "Hello" but not "World". This is likely related to how the data needs to be passed to `kn func invoke`.

To confirm that it's not the serverless function itself cutting off the data, you can run the following on the CLI (assuming you have `kn` and `kn-func` installed and on PATH):

```
❯ kn func invoke --data 'Hello World' --content-type application/json -t http://localhost:8080
```

expected output:
```
Context Attributes,
  specversion: 1.0
  type: function.output
  source: function
  id: 24c3af50-9e67-4b67-9acf-34d7a46253a4
  datacontenttype: application/json
Data,
  {
    "message": "Hello World"
  }
```

### PR DESCRIPTION
**Pull request by datho7561**
Fixes #3320

Signed-off-by: David Thompson <davthomp@redhat.com>

### CODE REVIEW

File: src/webview/openshift-terminal/openShiftTerminal.ts [hunk 1]
```diff
@@ -522,23 +522,23 @@ export class OpenShiftTerminalManager implements WebviewViewProvider {
         // wait until the webview is ready to receive requests to create terminals
         await this.webviewResolved;
 
-        const [cmd, ...args] = `${commandText}`.split(' ');
+        const tool = `${commandText}`.split(' ')[0];
```
Comment (msivasubramaniaan) [inline src/webview/openshift-terminal/openShiftTerminal.ts:525]: CRC setup failing for the mentioned path `C:\\Program Files\\Red Hat OpenShift Local\\crc.exe setup`

**Review by msivasubramaniaan — CHANGES_REQUESTED** [review]
Comment added

**Comment by msivasubramaniaan** [comment]
Related to https://github.com/redhat-developer/vscode-openshift-tools/issues/3337

File: src/webview/openshift-terminal/openShiftTerminal.ts (hunk 1 above)
Comment (datho7561) [reply src/webview/openshift-terminal/openShiftTerminal.ts]: It should work now if the path has spaces. I don't have access to a Windows computer currently, so please let me know if it works for you.

**Review by msivasubramaniaan — CHANGES_REQUESTED** [review]
Still facing the same issue.
image

**Comment by datho7561** [comment]
Okay, I tried something else. Could you please let me know if it works?

**Comment by msivasubramaniaan** [comment]
> Okay, I tried something else. Could you please let me know if it works?

It not helps. As discussed over slack replace with below code 

```
this._pty = (/\s/).test(this._file) ? ptyInstance.spawn(
                `${this._file}`,
                [
                    `${escapedArgs}`,
                ],
                this._options,
            ) : ptyInstance.spawn(
                'cmd.EXE',
                [
                    '/c',
                    `${this._file} ${escapedArgs}`,
                ],
                this._options,
            );
```

**Comment by datho7561** [comment]
Thanks! I've made that change!

**Comment by datho7561** [comment]
oops i forgot to add the changes when I pushed, should be good now

**Comment by msivasubramaniaan** [comment]
LGTM, Feel free to merge once CI done.

Thanks @datho7561

**Review by msivasubramaniaan — APPROVED** [review]

**Pull request merged**